src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -6860,15 +6861,33 @@ private bool IsUsingAliasInScope(string name)
                     // Support selecting an extension method from a type name in nameof(.)
                     return BindInstanceMemberAccess(node, right, boundLeft, rightName, rightArity, typeArgumentsSyntax, typeArguments, invoked, indexed, diagnostics);
                 }
-                else
+
+                CompoundUseSiteInfo<AssemblySymbol> useSiteInfo2 = GetNewCompoundUseSiteInfo(diagnostics);
+                this.LookupMembersWithFallback(lookupResult, leftType, rightName, rightArity, ref useSiteInfo2, basesBeingResolved: null, options: options);
+                diagnostics.Add(right, useSiteInfo2);
+                if (lookupResult.IsMultiViable)
                 {
-                    CompoundUseSiteInfo<AssemblySymbol> useSiteInfo = GetNewCompoundUseSiteInfo(diagnostics);
-                    this.LookupMembersWithFallback(lookupResult, leftType, rightName, rightArity, ref useSiteInfo, basesBeingResolved: null, options: options);
-                    diagnostics.Add(right, useSiteInfo);
-                    if (lookupResult.IsMultiViable)
+                    return BindMemberOfType(node, right, rightName, rightArity, indexed, boundLeft, typeArgumentsSyntax, typeArguments, lookupResult, BoundMethodGroupFlags.SearchExtensionMethods, diagnostics: diagnostics);
+                }
+
+                if (invoked)
+                {
+                    var boundMethodGroup = new BoundMethodGroup(
```

Comment:
> var boundMethodGroup = new BoundMethodGroup( [](http://example.com/codeflow?start=19&length=45)

Is body of this ```if``` a copy of portion of ```if (searchExtensionMethodsIfNecessary)``` from ```BindInstanceMemberAccess```? If so, consider extracting a common helper and reusing.